setup.yaml

```diff
@@ -0,0 +1,6 @@
+before_script:
```

Comment:
If you leave this as a before_script, can you name the file with something containing the word "node" or "npm"?